pkg/windows/windows.go

```diff
@@ -1008,6 +1014,21 @@ func (vm *windows) removeHNSNetwork(networkName string) error {
 	return nil
 }
 
+// enableWindowsFeatures enables the required Windows Container features on the Windows host.
+func (vm *windows) enableWindowsFeatures() error {
+	out, err := vm.Run(enableContainersCmd, true)
+	if err != nil {
+		return errors.Wrapf(err, "failed to enable required Windows features with output: %s", out)
+	}
+
+	// Reinitialize the SSH connection given enabling Windows features requires a VM restart
+	if err := vm.Reinitialize(); err != nil {
```

Comment:
will this reinitialize get restart the node even though the container feature is already enabled?

Reply:
It will not restart VM here, that's done in PS script. This is reinitializing SSH connection. But I think that is not required as well if the feature is already installed.